.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Armeria site
+
+on:
+  push:
+    tags:
+      - armeria-*
+
+env:
+  LC_ALL: 'en_US.UTF-8'
+
+jobs:
+  publish-site:
+    if: github.repository == 'line/armeria'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install svgbob_cli
+        run: |
+          sudo apt-get -y install cargo && cargo install svgbob_cli
+          echo "$HOME/.cargo/bin" >> $GITHUB_PATH
```

Comment:
Here it is😇
```suggestion
          echo '$HOME/.cargo/bin' >> $GITHUB_PATH
```

Reply:
Without double quotations (`"`), `$HOME` will be printed as it is. 😉
<img width="287" alt="image" src="https://user-images.githubusercontent.com/1866157/186108745-489b2133-b624-4ad1-a3d8-74eda6925582.png">